python/pyspark/sql/tests.py

```diff
@@ -56,6 +56,15 @@
 from pyspark.sql.utils import AnalysisException, ParseException, IllegalArgumentException
 
 
+_have_arrow = False
+try:
+    import pyarrow
+    _have_arrow = True
```

Comment:
I mean we should throw an exception when `useArrow` is used but no pyspark is installed.